### DISCUSSION TITLE
Cache for kratos cookie session

### DISCUSSION THREAD

**Discussion by dro-sh** [ory/oathkeeper#1083] [Q&A]
I have a kratos for identity management, oathkeeper for generation JWT tokens via id_token mutatior, my backend which expects token from oathkeeper with user info (like id, email, etc.). Oathkeeper have only authn for check cookie. All of this are placed in one kubernetes cluster. And also I have an SPA React app, which uses API from my backend.

I use this workflow: https://www.ory.sh/docs/oathkeeper/#access-control-decision-api. 
But as I understand all requests from frontend have next way (consider only successful authn):
<img width="670" alt="Screenshot 2023-03-29 at 10 22 50 AM" src="https://user-images.githubusercontent.com/37456151/228457305-77f70561-881d-481d-8273-67c5e308ab72.png">

Is there opportunity to cache (3) request from oathkeeper to kratos? If no ways and no plans to support it, how I can improve performance for this workflow (like place cdn or redis between oathkeeper or kratos)?

Also I found next question: https://github.com/ory/kratos/discussions/2098, but ttl works for generated token and cookie session has not ttl setting

**Comment by dro-sh** [comment]
@aeneasr any help for this, please?

**Reply by osbornk** [reply]
Did you ever figure out a solution for this? I am looking at this same issue.
